resources/src/commands/heart.rs

```diff
@@ -4,7 +4,7 @@ use slack_hook::{PayloadBuilder, Slack};
 
 // TODO Use reactions.add
 pub async fn run(channel: &str) {
-    let slack_webhook_url: String = crate::get_env_var(crate::DEVIL_BOT_TEST_CHANNEL_URL);
+    let slack_webhook_url: String = crate::get_env_var("DEVIL_BOT_TEST_CHANNEL_URL");
```

Comment:
Why did you do this?

Reply:
Clippy said it couldn't find it anymore after I refactored this. I can work through it again and make sure the imports are correct. I had gotten it to work this way so I left it.

Reply:
I see where I went wrong now. Fixed